src/SDKs/Blueprint/Management.Blueprint/Properties/AssemblyInfo.cs

```diff
@@ -11,8 +11,8 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.0.0")]
+[assembly: AssemblyVersion("0.9.1.0")]
```

Comment:
Please leave the `AssemblyVersion` unchanged, `AssemblyFileVersion` should be bumped